src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -1,17 +1,22 @@
 package edu.kis.vh.nursery;
 
-public class HanoiRhymer extends defaultCountingOutRhymer {
+public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-int totalRejected = 0;
+    public static final int empty = 0;
+    int totalRejected = empty;
```

Comment:
0 is ZERO! If there is no other meaning constant is redundant
totalRejected should be private